src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/ConsoleUpgrade.cs

```diff
@@ -145,7 +145,7 @@ private void ShowUpgradeSteps(IEnumerable<UpgradeStep> steps, IUpgradeContext co
                 _io.Output.WriteLine(step.Title);
                 nextStepFound = nextStepFound || step.Status != UpgradeStepStatus.Complete;
 
-                ShowUpgradeSteps(step.SubSteps, context, currentStep, offset + 1);
+                await ShowUpgradeStepsAsync(step.SubSteps, context, token, currentStep, offset + 1);
```

Comment:
Should this have a ConfigureAwait()?

Reply:
We've disabled it in this project since we know it doesn't have a synchronization context